views/partials/header.gohtml

```diff
@@ -8,7 +8,7 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <a class="navbar-brand" href="#">Contrast Go TestBench {{- with .Framework}} - {{ . }}{{end}}</a>
+      <a class="navbar-brand" href="/">Contrast Go TestBench {{- with .Framework}} - {{ . }}{{end}}</a>
```

Comment:
not sure what the intent was, but now you can click the banner and go back to the main page